### PR TITLE
#90_create_endpoint_for_entering_data_about_one_of_the_side

### PR DESCRIPTION
**Pull request by swem-st**

### CODE REVIEW

File: EasyCA.Backend.sln [hunk 1]
```diff
@@ -13,7 +13,7 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "ProtocolService.Data", "Pro
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "ProtocolService.Domain", "ProtocolService.Domain\ProtocolService.Domain.csproj", "{73CF0DD2-D4AB-4143-A75E-4B952D5A9406}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "ProtocolService.WebApi", "ProtocolService.WebApi\ProtocolService.WebApi.csproj", "{1D7624DC-3F3B-4995-9EA4-74F809A58043}"
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "ProtocolService.WebApi", "ProtocolService.WwebApi\ProtocolService.WebApi.csproj", "{F59BEE85-7D69-4BED-9338-664A56FBA0A2}"
```
Comment (ICHTLAY) [inline EasyCA.Backend.sln]: Fix wwebApi

File: ProtocolService.Data/Entities/Side.cs [hunk 2]
```diff
@@ -1,27 +1,25 @@
...
+        public string Email { get; set; }
+
         [BsonElement("TransportId")]
         [JsonProperty("TransportId")]
         public string TransportId { get; set; }
 
         [BsonElement("ListOfCircumstances")]
         [JsonProperty("ListOfCircumstances")]
-        public List<Circumstance> Circumstances { get; set; }
+        public List<string> Circumstances { get; set; }
```
Comment (ICHTLAY) [inline ProtocolService.Data/Entities/Side.cs]: change string to int

File: ProtocolService.Domain/ApiModel/RequestApiModels/EuroProtocolRequestModel.cs [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+using System;
+using System.Collections.Generic;
+
+namespace ProtocolService.Domain.ApiModel.RequestApiModels
+{
+    public class EuroProtocolRequestModel
+    {
+        public EuroProtocolRequestModel()
+        {
+            RegistrationDateTime = DateTime.Now;
```
Comment (ICHTLAY) [inline ProtocolService.Domain/ApiModel/RequestApiModels/EuroProtocolRequestModel.cs:11]: add 'IsClosed = false' to constructor

File: ProtocolService.Domain/ApiModel/RequestApiModels/SideRequestModel.cs [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+using ProtocolService.Data.Entities;
+
+namespace ProtocolService.Domain.ApiModel.RequestApiModels
+{
+    public class SideRequestModel
+    {
+        public string UserId { get; set; }
+        public string Email { get; set; }
+        public string TransportId { get; set; }
+        public List<string> Circumstances { get; set; }
```
Comment (ICHTLAY) [inline ProtocolService.Domain/ApiModel/RequestApiModels/SideRequestModel.cs]: change string to int

File: ProtocolService.Domain/Services/ServiceProtocol.cs [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+    {
+        private readonly ProtocolDbContext _context;
+        private readonly IMapper _mapper;
+
+        public ServiceProtocol(ProtocolDbContext context, IMapper mapper)
+        {
+            _context = context;
+            _mapper = mapper;
+        }
+
```
Comment (ICHTLAY) [inline ProtocolService.Domain/Services/ServiceProtocol.cs:23]: empty line

File: ProtocolService.WebApi/Controllers/EuroProtocolController.cs [hunk 6]
```diff
@@ -0,0 +1,29 @@
+﻿using Microsoft.Extensions.Configuration;
+using Microsoft.AspNetCore.Mvc;
+using ProtocolService.Domain.Interfaces;
+using System.Threading.Tasks;
+using ProtocolService.Domain.ApiModel.RequestApiModels;
+
+namespace ProtocolService.WwebApi
```
Comment (ICHTLAY) [inline ProtocolService.WebApi/Controllers/EuroProtocolController.cs]: wwebapi to webapi

**Review by ICHTLAY — CHANGES_REQUESTED** [review]

**Review by ICHTLAY — APPROVED** [review]

**Review by SashaKorniichuk — APPROVED** [review]

**Review by LizaShemetovska — APPROVED** [review]

**Pull request merged**